samza-sql/src/test/java/org/apache/samza/sql/system/TestAvroSystemFactory.java

```diff
@@ -291,6 +292,7 @@ private Object createComplexRecord(int index) {
       GenericRecord record = new GenericData.Record(ComplexRecord.SCHEMA$);
       record.put("id", index);
       record.put("string_value", "Name" + index);
+      record.put("bytes_value", java.nio.ByteBuffer.wrap(("sample bytes").getBytes()));
```

Comment:
s/java.nio.ByteBuffer.wrap/ByteBuffer.wrap 

Or, Are there naming conflicts with ByteBuffer class that require us to fully namespace it?

Reply:
Actually, nothing. I removed the namespace prefix.